src/detect-virt/detect-virt.c

```diff
@@ -159,6 +169,12 @@ static int run(int argc, char *argv[]) {
                         return log_error_errno(r, "Failed to check for user namespace: %m");
                 return !r;
 
+        case ONLY_CVM:
+                c = detect_confidential_virtualization();
```

Comment:
Currently, this does not return negative errno, but for consistency to other places, please handle the failure.